pnc-core/src/main/java/org/jboss/pnc/core/builder/DatastoreAdapter.java

```diff
@@ -40,7 +40,7 @@ public void storeResult(BuildTask buildTask, BuildResult buildResult) throws Dat
             RepositoryManagerResult repositoryManagerResult = buildResult.getRepositoryManagerResult();
 
             BuildRecord buildRecord = new BuildRecord();
-            buildRecord.setLatestBuildConfiguration(buildConfiguration);
+            buildRecord.setBuildConfigurationAudited(buildConfiguration.getBuildConfigurationAudited());
             // Build driver results
```

Comment:
Can be removed as it is done in setAuditDataToBuildRecord. I extracted it to method sa it is used also in case of storingResult with exception.